packages/core/src/modules/oob/OutOfBandApi.ts

```diff
@@ -76,6 +76,13 @@ export interface ReceiveOutOfBandInvitationConfig {
   reuseConnection?: boolean
   routing?: Routing
   acceptInvitationTimeoutMs?: number
+  isImplicit?: boolean
```

Comment:
Do you think we could split up the `receiveInvitation` method into a public and private one. The private one will allow for an `isImplicit`, but we don't add it to the public api. I don't think it's needed and you can already call the other method.